tests/functional/test.h

```diff
@@ -260,10 +260,12 @@ test_config_init(trunk_config           *splinter_cfg,  // OUT
 /*
  * Some command-line [config] arguments become test execution parameters.
  * Define a structure to hold these when parsing command-line arguments.
+ * This is shared by both functional and unit-test methods.
  */
 typedef struct test_exec_config {
    uint64 seed;
    uint64 num_inserts;
+   bool   verbose_progress; // --verbose-progress: During test execution
```

Comment:
I don't understand the distinction between this config and the more general `test_config`.

Reply:
`test_config` is a test-specific configuration, defined in `functional/splinter_test.h`.  [ I think it's probably mis-named, as it is not for *any* test, but just for this one very specific test. ]

This `test_exec_config` is a struct I had added earlier under SHA a2768f5, to address issue #272. This was done to support `--num-inserts` as a test-execution parameter.

The problem is that the existing `test_config` is only used by splinter_test, where as this new `--num-inserts` is used by unit-test `splinter_test`.

Right now, with this change, I am extending the test-execution config parameter set to include `--verbose-progress`, which currently will be used in the functional test. By adding it to this struct, it leaves open the possibility to do verbose-printing in unit-tests, too, in future.